pkg/asset/manifests/azure/cloudproviderconfig.go

```diff
@@ -12,6 +12,7 @@ type CloudProviderConfig struct {
 	CloudName                azure.CloudEnvironment
 	TenantID                 string
 	SubscriptionID           string
+	GroupName                string
```

Comment:
`ResourceGroupName` - Group is ambiguous as groups are an artefact or Azure AAD. Or match upstream `ResourceGroup` - https://github.com/kubernetes/kubernetes/blob/4b7c607ba47928a7be77fadef1550d6498397a4c/staging/src/k8s.io/legacy-cloud-providers/azure/azure.go#L86

Reply:
fixed in 
3932221